[2024-05-01T10:54:13.171+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: producer.update_dataset_2 scheduled__2024-04-30T00:00:00+00:00 [queued]>
[2024-05-01T10:54:13.181+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: producer.update_dataset_2 scheduled__2024-04-30T00:00:00+00:00 [queued]>
[2024-05-01T10:54:13.181+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2024-05-01T10:54:13.182+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2024-05-01T10:54:13.182+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2024-05-01T10:54:13.198+0000] {taskinstance.py:1383} INFO - Executing <Task(_PythonDecoratedOperator): update_dataset_2> on 2024-04-30 00:00:00+00:00
[2024-05-01T10:54:13.204+0000] {standard_task_runner.py:55} INFO - Started process 206 to run task
[2024-05-01T10:54:13.206+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'producer', 'update_dataset_2', 'scheduled__2024-04-30T00:00:00+00:00', '--job-id', '10', '--raw', '--subdir', 'DAGS_FOLDER/producer.py', '--cfg-path', '/tmp/tmp8dbfkgzx']
[2024-05-01T10:54:13.207+0000] {standard_task_runner.py:83} INFO - Job 10: Subtask update_dataset_2
[2024-05-01T10:54:13.266+0000] {task_command.py:376} INFO - Running <TaskInstance: producer.update_dataset_2 scheduled__2024-04-30T00:00:00+00:00 [running]> on host 3081978b9eea
[2024-05-01T10:54:13.338+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=producer
AIRFLOW_CTX_TASK_ID=update_dataset_2
AIRFLOW_CTX_EXECUTION_DATE=2024-04-30T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-04-30T00:00:00+00:00
[2024-05-01T10:54:13.339+0000] {python.py:177} INFO - Done. Returned value was: None
[2024-05-01T10:54:13.342+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/models/baseoperator.py:1390: RemovedInAirflow3Warning: Passing 'execution_date' to 'TaskInstance.xcom_push()' is deprecated.
  context['ti'].xcom_push(key=key, value=value, execution_date=execution_date)

[2024-05-01T10:54:13.364+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=producer, task_id=update_dataset_2, execution_date=20240430T000000, start_date=20240501T105413, end_date=20240501T105413
[2024-05-01T10:54:13.418+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2024-05-01T10:54:13.436+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
